modules/scala/jupyter-api/src/main/scala/almond/display/Image.scala

```diff
@@ -1,14 +1,17 @@
 package almond.display
 
-import java.io.{ByteArrayInputStream, IOException}
+import java.awt.image.RenderedImage
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, IOException}
 import java.net.{HttpURLConnection, URL, URLConnection}
 import java.util.Base64
 
+import javax.imageio.ImageIO
+
 import scala.util.Try
 
 final class Image private (
-  val width: Option[String],
```

Comment:
Why changing the type from `Int` to `String`? `String` allows to pass relative dimensions in particular.

Reply:
Ah yes that doesn't make any sense. For some reason I had in mind that the img width attribute has to be just a number, which is of course completely wrong. I'll change it back.